### PR TITLE
Add OpInfo entry for as_strided_copy

### PR DESCRIPTION
**Pull request by rec**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #127231

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire

### CODE REVIEW

**Review by lezcano — COMMENTED** [review]
needs to be added to the decompositions/__init__ list.

File: torch/_decomp/__init__.py [hunk 1]
```diff
@@ -264,6 +264,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.aminmax,
             aten.arange.default,
             aten.arange.start,
+            aten.as_strided_copy,
```
Comment (lezcano) [inline torch/_decomp/__init__.py:267]: So, apparently inductor doesn't like `as_strided`, so we should not add it to this list. We then should remove it from this list (not decompose it, I guess). See if inductor handles as_strided at all, if it does, we should handle this one te same way than `as_strided`, if not, then we should just not decompose it and let inductor fallback to eager.

Comment (rec) [reply torch/_decomp/__init__.py:267]: I did manage to get the inductor tests to work for everything except `torch.bool`, which I could not puzzle out as to why it didn't work, so I `skip`ped it.

Comment (rec) [reply torch/_decomp/__init__.py:267]: Well, that was a false claim, it fails for all types. Not sure why I thought otherwise. Updating coming.

**Review by lezcano — APPROVED** [review]
Sounds good, thank you! (modulo CI)

**Comment by rec** [comment]
FYI, the errors are because I'm skipping tests that now succeed because the `as_strided_copy` is marked as not being decomposed. I'm fixing these...

**Pull request closed without merging**